requirements/init_script.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+pip install -r requirements/requirements.txt
+
+# create docker container
+docker run -d \
+--name fate_server \
+-p 3850:5432 \
+-v $HOME/postgresql/fate_server:/var/lib.postgresql/fate_server \
+-e POSTGRES_PASSWORD=12345F \
+-e POSTGRES_USER=app \
+-e POSTGRES_DB=phrases_db \
+postgres
+sleep 2
+
+./requirements/setup_env.py
```

Comment:
it wouldnt work, because:
1) you do not give permission to execute with chmod (again, did you even try your own setup script?)
2) python script should be either ran with python3, either it should have shabang line with path to python interpreter inside.
Neither of these are present in your scripts.